modules/nw-throughput-troubleshoot.adoc

```diff
@@ -1,50 +1,39 @@
-reveal any cause of the problem:
+Sometimes applications deployed through {product-title} can cause network throughput issues such as unusually high latency between specific services.
 
-* Use a packet analyzer, such as ping or link:http://www.tcpdump.org/[tcpdump]
-to analyze traffic between a pod and its node.
+Use the following methods to analyze performance issues if pod logs do not reveal any cause of the problem:
+
+* Use a packet analyzer, such as ping or tcpdump to analyze traffic between a pod and its node.
 +
-For example, run the tcpdump tool on each pod while reproducing the behavior
-that led to the issue.
-Review the captures on both sides to compare send and receive timestamps to
-analyze the latency of traffic to and from a pod.
-Latency can occur in {product-title} if a node interface is overloaded with
-traffic from other pods, storage devices, or the data plane.
+For example, link:https://access.redhat.com/solutions/4569211[run the tcpdump tool on each pod] while reproducing the behavior that led to the issue. Review the captures on both sides to compare send and receive timestamps to analyze the latency of traffic to and from a pod. Latency can occur in {product-title} if a node interface is overloaded with traffic from other pods, storage devices, or the data plane.
 +
 [source,terminal]
 ----
 $ tcpdump -s 0 -i any -w /tmp/dump.pcap host <podip 1> && host <podip 2> <1>
 ----
 +
-<1> `podip` is the IP address for the pod. Run the `oc get pod <pod_name> -o wide` command to get
-the IP address of a pod.
+<1> `podip` is the IP address for the pod. Run the `oc get pod <pod_name> -o wide` command to get the IP address of a pod.
 +
-tcpdump generates a file at `/tmp/dump.pcap` containing all traffic between
-these two pods. Ideally, run the analyzer shortly
-before the issue is reproduced and stop the analyzer shortly after the issue
-is finished reproducing to minimize the size of the file.
-You can also run a packet analyzer between the nodes (eliminating the SDN from
-  the equation) with:
+tcpdump generates a file at `/tmp/dump.pcap` containing all traffic between these two pods. Ideally, run the analyzer shortly before the issue is reproduced and stop the analyzer shortly after the issue is finished reproducing to minimize the size of the file. You can also link:https://access.redhat.com/solutions/5074041[run a packet analyzer between the nodes] (eliminating the SDN from the equation) with:
 +
 [source,terminal]
 ----
 $ tcpdump -s 0 -i any -w /tmp/dump.pcap port 4789
 ----
 
-* Use a bandwidth measuring tool, such as iperf, to measure streaming throughput
-and UDP throughput. Run the tool from the pods first, then from the nodes,
-to locate any bottlenecks.
+* Use a bandwidth measuring tool, such as link:https://access.redhat.com/solutions/6129701[iperf], to measure streaming throughput and UDP throughput. Run the tool from the pods first, then from the nodes, to locate any bottlenecks.
```

Comment:
```suggestion
* Use a bandwidth measuring tool, such as link:https://access.redhat.com/solutions/6129701[`iperf`], to measure streaming throughput and UDP throughput. Locate any bottlenecks by running the tool from the pods first, and then running it from the nodes.
```
Does UDP need to be expanded?

Reply:
It is mostly kept abbreviated in the docs, I think in many cases UDP is more recognizable than what it's abbreviating 😄 (user diagram protocol). I think customers in this part of the docs are familiar with networking principles e.g. TCP/UDP/IP